todo_app/app.py

```diff
@@ -1,15 +1,55 @@
-from flask import Flask
-
+from flask import Flask,request,render_template,redirect,url_for,flash
+from todo_app.data.session_items import get_items,add_item,get_item,save_item,delete_item
+from operator import itemgetter
 from todo_app.flask_config import Config
 
 app = Flask(__name__)
 app.config.from_object(Config)
 
 
-@app.route('/')
+
+# Call to add an item or display the page
+@app.route('/', methods=['GET', 'POST'])
 def index():
-    return 'Hello World!'
+    listofitems = get_items()
+    
+
+    sortedListofitems = sorted(listofitems, key=itemgetter('status') )
+
+    
+    if request.method == 'POST':
+        title = request.form.get('itemname')
+        add_item(title)
+        ##return  render_template('index.html',getlist = listofitems)
+        return redirect(url_for('index'))
```

Comment:
It would be nicer to pull out a separate function/`@app.route` for adding items. It shares zero logic with displaying the index page.